__tests__/units/colors.test.ts

```diff
@@ -0,0 +1,12 @@
+import { describe, expect, it } from '@jest/globals';
+import { getColor } from '@/colors';
+
+describe('colors', () => {
+    it('should return a valid color for index-arguments from 0 to 99', () => {
+        expect.hasAssertions();
+        for (let i = 0; i <= 100; i++) {
+            const color = getColor(i);
+            expect(color).not.toHaveLength(0);
```

Comment:
Can you make this assertion "harder"? As far as I can see, all colors returned by getColor should still be in hex notation, which we can definitely check for.

Reply:
The bootstrap variables get resolved at runtime, so it isn't as easy. I will see what can be done

Reply:
There isn't really a way to test if the variables actually resolve to colours in the document. Such functionality is explicitly out of scope of our testing libraries. However, I've made the test stricter by checking for a valid CSS variable syntax, at least.